mlfinlab/backtest_statistics/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Functions derived from Chapter 14: Backtest Statistics
```

Comment:
Let's use a better description. Explain the module and what code is included in it. I want to stress the depth I desire and how important these docstrings are. They are almost more important than the code written. They form the basis for how users will make use of your code.

Reply:
Will a list of functions and their short descriptions (as comments in statistics.py) be enough? Or a shorter description (a sentence/function) is preferred?

Reply:
Let's have the focus be on what this module (file) is about. Eventually, this package will include authors other than de Prado, thus a module should have a purpose. This docstring should highlight that purpose. 

I wouldn't do a list of functions because then we need to update the docstring as we add new functions.